Raven.Client.Lightweight/Connection/ReplicationInformer.cs

```diff
@@ -140,26 +155,34 @@ protected override string GetServerCheckUrl(string baseUrl)
 
         public void RefreshReplicationInformation(AsyncServerClient serverClient)
         {
-            RefreshReplicationInformationInternal(serverClient.Url, () => AsyncHelpers.RunSync(() => serverClient.DirectGetReplicationDestinationsAsync(new OperationMetadata(serverClient.Url, serverClient.PrimaryCredentials, null), null)));
+            AsyncHelpers.RunSync(() =>
+                RefreshReplicationInformationInternalAsync(serverClient.Url, () => serverClient.DirectGetReplicationDestinationsAsync(new OperationMetadata(serverClient.Url, serverClient.PrimaryCredentials, null), null)));
         }
 
         public override void RefreshReplicationInformation(ServerClient serverClient)
         {
-            RefreshReplicationInformationInternal(serverClient.Url, () => serverClient.DirectGetReplicationDestinations(new OperationMetadata(serverClient.Url, serverClient.PrimaryCredentials, null), null));
+            AsyncHelpers.RunSync(() =>
+                RefreshReplicationInformationInternalAsync(serverClient.Url, () => Task.FromResult(serverClient.DirectGetReplicationDestinations(new OperationMetadata(serverClient.Url, serverClient.PrimaryCredentials, null), null))));
```

Comment:
Wasn't entirely sure here whether you would prefer to offload `serverClient.DirectGetReplicationDestinations` or not. Given the sync nature of this code I went straight to not offload. Please advise